docs/release-notes.md

```diff
@@ -1,3 +1,12 @@
+## 0.13.8
+
+* Revert `Queue(maxsize=1)` fix for `BaseHTTPMiddleware` middleware classes and streaming responses. 
+  * Before version `0.13.7`, [some users](https://github.com/encode/starlette/issues/1022) were relying on behavior where the response object would be fully evaluated within their middleware based on `BaseHTTPMiddleware`. (Of course, this was a problem for `StreamingResponse`s with this middleware, because they were also being loaded fully into memory.) The `maxsize=1` fix released in version `0.13.7` explicitly prevented the response object from being fully evaluated until later `await` calls, but any users who were dropping this response for any other created inside their middleware were likely to see pending tasks accumulate. 
```

Comment:
Thoughts on dropping the mdr verbose details? What's important here seems to be that the fix corrected a regression that appeared in 0.13.7. I don't think the nitty gritty technical details are of importance to users yet - we can keep this in our heads for when we take a more thorough review at the state of concurrency and streaming. In other words: should we drop this bullet point? WDYT?

Reply:
Sounds good. All of this discussion should be present in the PR for anyone who wants it. I'll drop it from here.